examples/agones-xonotic-xds/proxy-pool.yaml

```
@@ -42,8 +42,8 @@ spec:
     spec:
       containers:
         - name: quilkin
-          image: us-docker.pkg.dev/quilkin/release/quilkin:0.4.0
-          args: ["run", "--management-server", "http://quilkin-manage-agones:80"]
+          image: us-docker.pkg.dev/quilkin/release/quilkin:0.5.0-dev
```

```suggestion
          image: us-docker.pkg.dev/quilkin/release/quilkin:0.5.0
```

That way I don't need to remember to go back and fix it at release time.